dev-util/radare2/radare2-1.1.0.recipe

```diff
@@ -93,11 +98,6 @@ INSTALL()
 {
 	make install
 	cp libr/*/libr_*.so $libDir
-
-	mkdir -p `dirname $docDir`
-	mv $prefix/share/doc $docDir
-	rm -rf $prefix/share
```

Comment:
I don't think this is fixed upstream, Line 98 causes the build to fail in 1.1.0  when moving the docs to $docDir.